src/index.ts

```diff
@@ -3,20 +3,20 @@ import { BIP32 } from './bip32'
 import { Keypair } from './keypair'
 import { hmacSHA512 } from './crypto'
 
+export * from './keypair'
+
 /**
  * A derived Wallet account
  * @property index The index for the given account
  * @property seed The Ed25519 private seed/key
  * @property address The Ed25519 public key
  */
 export interface Account {
-  index: number
+  index: number | number[]
```

Comment:
We now support multi-dimensional indices, but by default, will still support a single scalar index.

Reply:
very cool